nab/detectors/htmcore/htmcore_detector.py

```diff
@@ -331,65 +326,74 @@ def modelRun(self, ts, val):
           # print(self.tm_info)
           pass
 
-      if PANDA_VIS_ENABLED:
-          pandaServer.BlockExecution()
-
+      # 6. panda vis
+      if PANDA_VIS_BAKE_DATA:
+          # ------------------HTMpandaVis----------------------
+          # fill up values
+          pandaBaker.inputs["Value"].stringValue = "value: {:.2f}".format(val)
```

Comment:
could you, please, add a short readme (and comments here) what is done in terms of setting up the Viz. Imagine the user has a different inputs/network a bit. Or a link to readme where this is explained. 

What are inputs, layers, dataStreams, ... ? 

With that, this is good to merge. Thank you.

Reply:
ok done